### PR TITLE
Data/props being passed through components & corresponding tests

### PR DESCRIPTION
**Pull request by ZeMunchkin**
Trello Ticket: 
https://trello.com/c/Ov80d5FL/11-l-pass-data-props-through-components-classes-and-write-tests

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-
+      reviews: [],
+      restaurant: 'string',
+      restId: 'number',
```
Comment (MichaelDurfey) [inline client/components/app.jsx]: if you're expecting a number here I'd suggest having a number as a placeholder

File: tests/clientTests/componentsTests/button.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Button from '../../../client/components/button';
+
+describe('Button container', () => {
+  const button = mount(<Button id="456" restaurant={123} score={4} type="Cool" />);
+
+  test('Button should be stateful', () => {
+    expect(button).toHaveProperty('state');
```
Comment (MichaelDurfey) [inline tests/clientTests/componentsTests/button.test.js]: This is testing whether button has a property of state? That doesn't seem to be a property on button from what I see here.

File: client/components/app.jsx (hunk 1 above)
Comment (ZeMunchkin) [reply client/components/app.jsx]: Herp derp. Thanks :)

File: tests/clientTests/componentsTests/button.test.js (hunk 2 above)
Comment (ZeMunchkin) [reply tests/clientTests/componentsTests/button.test.js]: It's just a test to make sure I made buttons stateful (using this.state) and not a regular component. Will be more important later when I add button functionality.

**Pull request merged**